### PR TITLE
Feat: Add ConfirmPopover component

### PR DESCRIPTION
**Pull request by HugoPerard**
Add new ConfirmPopover component, with docs stories and translations

Capture d’écran 2022-03-18 à 22 04 26

### CODE REVIEW

File: src/locales/en/components.json [hunk 1]
```diff
@@ -8,6 +8,10 @@
   "confirmMenuItem": {
     "confirmText": "Click to confirm"
   },
+  "confirmPopover": {
+    "confirmText": "Confirm",
+    "heading": "Are you sure ?"
```
Comment (ivan-dalmet) [inline src/locales/en/components.json:13]:
```suggestion
    "heading": "Are you sure?"
```

**Review by ivan-dalmet — APPROVED** [review]

File: src/components/ConfirmPopover/docs.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import { ConfirmPopover } from '.';
+
+export default {
+  title: 'Components/ConfirmPopover',
+};
+
+export const Default = () => {
+  return (
+    <>
```
Comment (yoannfleurydev) [inline src/components/ConfirmPopover/docs.stories.tsx:13]: No need for Fragment here.

File: src/components/ConfirmPopover/docs.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    </>
+  );
+};
+
+export const WithCustomParameters = () => {
+  return (
+    <>
+      <ConfirmPopover
+        title="Popover Title"
+        message={'Custom message'}
```
Comment (yoannfleurydev) [inline src/components/ConfirmPopover/docs.stories.tsx:26]:
```suggestion
        message="Custom message"
```

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]

**Comment by HugoPerard** [comment]
@yoannfleurydev Your feedbacks are solved :)

**Pull request merged**